org.fdroid.Repomaker.json

```diff
@@ -0,0 +1,260 @@
+                    "sha256": "6b4b3ba2253d84ed3771c8050728d597c91cfce898713beb7b64a305b6f11aad"
+                }
+            ]
+        },
+        {
+            "name": "git",
+            "no-autogen": true,
+            "sources": [{
+                "type": "archive",
+                "url": "https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.9.5.tar.gz",
+                "sha256": "8fa575338137d6d850b52d207cf7155cd1f4003ebd698f0fb75f65efb862ef7f"
+            }]
+        },
+        {
+            "name": "PyQt5",
+            "cleanup": ["/bin/sip", "/include", "/lib/python3.5/site-packages/*.pyi"],
+            "config-opts": ["--disable-static", "--enable-x11"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.11.2/PyQt5_gpl-5.11.2.tar.gz",
+                    "sha256": "7caa581155c3433716b7e6aba71fe1378cd7d92f4155c266d60e5cffb64e9603"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "python3 configure.py --confirm-license --no-docstrings --assume-shared --no-sip-files --no-qml-plugin --no-tools --no-qsci-api -d ${FLATPAK_DEST}/lib/python3.5/site-packages --sip=${FLATPAK_DEST}/bin/sip --sip-incdir=${FLATPAK_DEST}/include --stubsdir=${FLATPAK_DEST}/lib/python3.5/site-packages --disable=QtSensors --disable=QtQuick --disable=QtQml --disable=QtTest --disable=QtQuickWidgets --disable=QtSql --disable=QtXmlPatterns --disable=QtMultimedia --disable=QtMultimediaWidgets --disable=QtLocation --disable=QtDesigner --disable=QtOpenGL --disable=QtBluetooth --disable=QtNfc --disable=QtPositioning"
+                    ]
+                }
+            ],
+            "no-autogen": true,
+            "modules": [{
+                "name": "sip",
+                "no-autogen": true,
+                "sources": [
+                    {
+                        "type": "archive",
+                        "url": "https://sourceforge.net/projects/pyqt/files/sip/sip-4.19.12/sip-4.19.12.tar.gz",
+                        "sha256": "24617fc31b983df075500ecac0e99d2fb48bf63ba82d4a17518659e571923822"
+                    },
+                    {
+                        "type": "shell",
+                        "commands": [
+                            "python3 configure.py --sip-module PyQt5.sip -b ${FLATPAK_DEST}/bin -d ${FLATPAK_DEST}/lib/python3.5/site-packages -e ${FLATPAK_DEST}/include -v ${FLATPAK_DEST}/share/sip --stubsdir=${FLATPAK_DEST}/lib/python3.5/site-packages"
+                        ]
+                    }
+                ]
+            }]
+        },
+        "python-deps.json",
+        "fdroidserver-deps.json",
+        {
+            "name": "fdroidserver",
+            "buildsystem": "simple",
+            "build-commands": [
+                "git apply fdroidserver-0.8-allow-flatpak-aapt.patch",
```

Comment:
Is git used only to apply this patch? Or it's required by the app itself? You could use a source of type patch

Reply:
No, git is required by Repomaker to push repos to git repositories at GitLab/GitHub/whatsoever.

See the third step of "Step 3: Customize Your Repo" at https://f-droid.org/tutorials/create-repo/ for an example.

Reply:
But I could indeed use a source of type patch. Will do this.

Reply:
Makes sense. Sorry for my "stupid" question, had no idea how Repomaker works. Thanks!

Reply:
No need to be sorry. Thanks a lot for your quick review!